### PR TITLE
hubspot-batching: add json array support

### PR DESCRIPTION
**Pull request by arajguha**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by akashrpo** [comment]
we need to add `"JSON_ARRAY"` in `versioned_batch_braze_input.json` and `hs_router_input.json` ?

File: __tests__/data/hs_router_input.json [hunk 1]
```diff
@@ -161,11 +161,11 @@
...
-              "value": "testhubspot2@email.com"
+              "value": "testhubspot3@email.com"
             },
             {
               "property": "firstname",
-              "value": "Test Hubspot"
+              "value": "Test Hubspot3"
             }
           ]
         },
```
Comment (akashrpo) [inline __tests__/data/hs_router_input.json:171]: we need to add `"JSON_ARRAY": {},` below ?

Comment (arajguha) [reply __tests__/data/hs_router_input.json:171]: @utsabc  @akashrpo  how are the tests passing if its missing?

Comment (akashrpo) [reply __tests__/data/hs_router_input.json:171]: It is in the *_input.json part and we are missing the new structure here and while extraction we are extracting from JSON.properties hence, getting the desired result ?

**Review by sanpj2292 — APPROVED** [review]

**Pull request merged**